lib/ex_aws/ec2.ex

```diff
@@ -21,7 +21,7 @@ defmodule ExAws.EC2 do
   ExAws.EC2.describe_instances(filters: [image_id: "ami-1ecae776"])
   ExAws.EC2.describe_instances(filters: ["network-interface.availability-zone": "us-east-1a"])
   ExAws.EC2.describe_instances(filters: ["tag:elasticbeanstalk:environment-name": "demo"])
-  ExAws.EC2.describe_instance_status(instances: ["i-e5974f4c"])
+  ExAws.EC2.describe_instance_status(instance_id: ["i-e5974f4c"])
```

Comment:
Just from a quick glance, what you currently have looks good to me. 

@benwilson512, your thoughts?